assets/pages/paper-subscription-landing/components/content.scss

```diff
@@ -0,0 +1,3 @@
+.paper-subscription-landing-content__focusable {
+  outline: 0;
+}
```

Comment:
Could this live in the `paperSubscriptionLandingPage.scss` stylesheet?

Reply:
Any particular reason? I might be too used to CSS modules but like the idea of keeping it as close as possible to where this behaviour happening to the code, even if it's a three liner